### DISCUSSION TITLE
App Privacy Details / AppStoreConnect API

### DISCUSSION THREAD

**Discussion by tmaly1980** [fastlane/fastlane#18428] [Q&A]
Apple is now enforcing privacy details in app submissions. Fastlane's upload_app_privacy_details_to_app_store unfortunately uses iTunes Portal, which requires 2FA at this point. These actions don't seem to support Apple's AppStoreConnect API yet. Is there a way to get AppStoreConnect API's token to authenticate for the iTunes Portal client? 

Or is there a new API endpoint that allows for updating the app privacy details?

**Comment by dinsen** [comment]
Hi @tmaly1980 
Currently there is no endpoint for this, so for now you need to use the session login.
